src/utils/createAlertsContent.js

```diff
@@ -162,13 +162,17 @@ export const createAlertRowData = ({ name, ...alert }) => {
         tooltip: name,
         type: 'link'
       },
-      {
-        id: `projectName.${alert.id}`,
-        headerId: 'projectName',
-        headerLabel: 'Project name',
-        value: alert.project,
-        className: 'table-cell-1'
-      },
+      ...(isCrossProjects
```

Comment:
better to add a hidden property, so when we'll have some other condition to show or not some filter we can use this prop to display it

Reply:
@mariana-furyk added hidden property.

Reply:
but I don't see that you're using it here? still checking for isCrossProjects and returning result based on it. you should also add hidden:  isCrossProjects  here and remove check for  isCrossProjects above

Reply:
@mariana-furyk  the isCrossProjects state is currently used in both the AlertsTable and AlertsFilters components.

In AlertsFilters, it removes the "Projects" option in the filter popup when the screen is in a "per-project" view.

If we remove the isCrossProjects check from the Alert.js component, what alternative approach would you suggest for passing the isCrossProjects state to the AlertsFilters component?
thanks